addons/mail/static/src/discuss/voice_message/common/voice_recorder.scss

```diff
@@ -0,0 +1,8 @@
+.o-mail-VoiceRecorder-dot {
+  animation: flash 2s ease infinite;
+}
+
+.o-mail-VoiceRecorder-elapsed {
+  transition: all 1s ease-in-out;
+  width: max-content;
```

Comment:
Animation is not smooth. Instead of using `all` that can trigger more animations than intended, better just target `max-width`, and let's use 2s so it's right not too long and not too short.
Also the animation should start at size 0. To do so, we can keep the element in UI at 0 pixel, and make the transition from 0 to content size.